### ISSUE TITLE
这个代码是完整的吗

### ISSUE THREAD

**Issue by 526222883** [NLP-Playground/LaSS#1]
请问这个是完整的代码吗

**Comment by linzehui** [comment]
code是完整的。需要安装fairseq才能使用。

**Comment by TPFRL** [comment]
@linzehui  Could you release the config file you used?

**Comment by linzehui** [comment]
@TPFRL 

Recently I'm too busy to release all the configs and models. Here are three yaml configs for you to refer.  
[configs.zip](https://github.com/NLP-Playground/LaSS/files/7811804/configs.zip)

1. Train the vanilla multilingual baseline.
`bash scripts/train.sh —config baseline.yml`

2. Fine-tune the baseline for each language pair
`bash scripts/train.sh —config finetune.yml`

3. Obtain the masks
`python3 toolbox/generate_mask.py —checkpoint-path xx —mask-path /path/to/destination —gen-mask-with-prob —mask-prob 0.x —gen-part —exclude-output-proj`

4. Continue training the vanilla multilingual model with the obtained masks
`bash scripts/train.sh —config multilingual.yml`

**Comment by jinbo18101206396** [comment]
代码执行到第四步有问题
